src/slackify/slack.py

```diff
@@ -41,11 +43,11 @@ def text_block(text: str, markdown=True):
     }
 
 
-def block_reply(blocks: list):
+def block_reply(blocks: list) -> Tuple[str, int, JT]:
```

Comment:
In Python's typing, dict doesn't exist. `Dict` is what should be use and types should be specified. I only put it as its own type to avoid writing the same return type over and over.

Reply:
Yeah, `typing.Dict` will also work and would make it easier if we want to specify more guarantees about its keys or values.
Choose whatever you like the most.

Reply:
Plus, JT is actually that dictionary with `'content-type': 'application/json'`. I think Dict`[str, str]` suits it very well but I'll leave it as Dict only then.